extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/ImplicitValueResolverBuildItem.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.qute.deployment;
+
+import org.jboss.jandex.ClassInfo;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class ImplicitValueResolverBuildItem extends MultiBuildItem {
+
+    private final ClassInfo clazz;
```

Comment:
> I think we tried to avoid that.

Why? I'm pretty sure we do propagate jandex elements in many build items and since they're immutable it should be fine.